config/envs/rhpam-authoring-ha.yaml

```diff
@@ -577,4 +577,3 @@ others:
           port:
             targetPort: rest
 ## ES/AMQ END
-
```

Comment:
Please add back the newline at the end of the file, its absence causes diff errors and the like on some systems.

Reply:
I was able to fix this one.